mantis-shaded/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Netflix, Inc.
```

Comment:
2020. You can add in copyright from the contributing directions https://github.com/Netflix/mantis/blob/master/CONTRIBUTING.md.